ynr/apps/parties/tests/test_party_choices.py

```diff
@@ -45,16 +45,14 @@ def test_enough_candidates_in_current_election(self):
         self.create_lots_of_candidates(
             self.election, ((self.ld_party, 30), (self.green_party, 15))
         )
+
         party_choices = Party.objects.register("GB").party_choices()
         self.assertEqual(
             party_choices,
             [
                 (u"", u""),
-                (
-                    str(self.ld_party.ec_id),
-                    u"Liberal Democrats (30 candidates)",
-                ),
-                (str(self.green_party.ec_id), u"Green Party (15 candidates)"),
+                (str(self.ld_party.ec_id), u"Liberal Democrats"),
+                (str(self.green_party.ec_id), u"Green Party"),
```

Comment:
It looks like these tests are written using `str()`, `u""` for python 2/3 compatibility. While you're modifying these files, can we de-clutter some of this to just use strings so its a bit more readable?